core/client/client.go

```diff
@@ -127,7 +127,7 @@ func New(ktClient pb.KeyTransparencyClient,
 		Verifier:    ktVerifier,
 		cli:         ktClient,
 		DirectoryID: directoryID,
-		mutate:      entry.MutateFn,
+		mutate:      entry.ReduceFn,
```

Comment:
nit: the "mutate" property of the struct would be clearer if named "reducer", or similar.